### DISCUSSION TITLE
Multi Execution Mode

### DISCUSSION THREAD

**Discussion by josepcorrea** [Eugeny/tabby#6622] [Q&A] [answered]
Hello,

This mode allows you to display all your terminals at the same time and to write your commands to all of them at once: when you press a keyboard key, it will write the same letters on all your terminals at once.

¿This option exists?

Thx

**Comment by Eugeny** [comment, marked as answer]
Yes - see "broadcast", "combine" and "explode" here: https://github.com/Eugeny/tabby/discussions/5259
